code/game/objects/structures/displaycase.dm

```diff
@@ -44,8 +44,7 @@
 	update_icon()
 
 /obj/structure/displaycase/gooncode/setContent()
-	occupant=new /obj/item/toy/gooncode(src)
-	desc = "The glass is cracked and there are traces of something leaking out."
```

Comment:
Should change this to "It looks like something has fully leaked out." now

Reply:
I'll put that description directly on the gooncode, the case itself now contains the pomfcoin